tests/Unit/API/Site/Controllers/MerchantCenter/PhoneNumberControllerTest.php

```diff
@@ -0,0 +1,158 @@
+		$response = $callback( new WP_REST_Request( 'GET', self::ROUTE ) );
+
+		$this->assertInstanceOf( WP_REST_Response::class, $response );
+		$this->assertEquals( $error_code, $response->get_status() );
+		$data = $response->get_data();
+		$this->assertArrayHasKey( 'message',  $data );
+		$this->assertEquals( $this->get_account_exception()->getMessage(), $data['message']);
+	}
+
+	public function test_get_phone_number_endpoint_edit_callback() {
+		$this->merchant_verification->expects( $this->any() )
+									->method( 'update_phone_number' )
+									->willReturn(  $this->valid_account_phone_number );
+
+		$this->options->expects( $this->any() )
+					  ->method( 'get_merchant_id' )
+					  ->willReturn(  $this->valid_account_id );
+
+		$callback = $this->get_phone_number_controller_reflection_method( 'get_phone_number_endpoint_edit_callback' )
+						 ->invokeArgs($this->phone_number_controller,[]);
+
+		$request = new WP_REST_Request( 'POST', self::ROUTE );
+		$request->set_header( 'content-type', 'application/json' );
+		$request->set_body( '{"phone_number":"' . $this->valid_account_phone_number . '"}' );
+
+		$this->validate_success_response( $callback( $request ) );
+	}
+
+	public function test_get_phone_number_endpoint_edit_callback_exception() {
+		$error_code = 401;
+		$this->merchant_verification->expects( $this->any() )
+									->method( 'update_phone_number' )
+									->willThrowException( $this->get_account_exception( $error_code ) );
+
+		$callback = $this->get_phone_number_controller_reflection_method( 'get_phone_number_endpoint_edit_callback' )
+						 ->invokeArgs($this->phone_number_controller,[]);
+
+		$request = new WP_REST_Request( 'POST', self::ROUTE );
+		$request->set_header( 'content-type', 'application/json' );
+		$request->set_body( '{"phone_number":"' . $this->valid_account_phone_number . '"}' );
+
+		/** @var WP_REST_Response $response */
+		$response = $callback( $request );
+
+		$this->assertInstanceOf( WP_REST_Response::class, $response );
+		$this->assertEquals( $error_code, $response->get_status() );
+		$data = $response->get_data();
+		$this->assertArrayHasKey( 'message',  $data );
+		$this->assertEquals( $this->get_account_exception()->getMessage(), $data['message']);
+	}
+
+	/**
+	 * Validate that the response is success and contains the expected test data.
+	 *
+	 * @param WP_REST_Response $response
+	 */
+	protected function validate_success_response( WP_REST_Response $response ) {
+		$this->assertInstanceOf( WP_REST_Response::class, $response );
+		$this->assertEquals( 200, $response->get_status() );
+		$data = $response->get_data();
+		$this->assertArrayHasKey( 'phone_number',  $data );
+		$this->assertEquals( $this->valid_account_phone_number, $data['phone_number']);
+		$this->assertArrayHasKey( 'id', $data );
+		$this->assertEquals( $this->valid_account_id, $data['id']);
+	}
+
+	protected function get_phone_number_controller_reflection_method( string $method ): ReflectionMethod {
```

Comment:
There are other tests here in this class that apply to protected methods. I think it's best to remove those as well and implement REST testing for their functionality in your upcoming PR.